Server/mods/deathmatch/logic/CGame.cpp

```diff
@@ -2443,6 +2443,10 @@ void CGame::Packet_Bulletsync(CBulletsyncPacket& Packet)
     CPlayer* pPlayer = Packet.GetSourcePlayer();
     if (pPlayer && pPlayer->IsJoined())
     {
+        // Early return when the player attempts to fire a weapon they do not have
```

Comment:
I think it would be nice to add a comment explaining what could cause this (Cheaters, etc).